.github/workflows/pr-lint.yml

```
@@ -0,0 +1,34 @@
+name: Pull Request Lint
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
+
+jobs:
+  validate:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-pr-lint@v1
+        env:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          scopes: |-
+            compiler
+            grammar
+            docs
+          types: |-
+            feat
+            fix
+            docs
+            chore
+            revert
+            build
+          subjectPattern: ^[^A-Z][^:]+[^.]$
```

```suggestion
          subjectPattern: ^[^:]+$
```

I'm not sure I care about the capitalization nor the ending with a period. Thoughts?

---

for me its consistency. When looking through release notes id prefer to see the conventions be consistent then you never have someone say "When should I use Feat vs feat"

---

This is for just the subject though. The scope and type are still controlled.

---

Yea you are right. I still think there is a consistency thing here but lets let hell break loose 😁